### PR TITLE
Add type to typescript knexfile

### PR DESCRIPTION
**Pull request by tharrison4815**

### CODE REVIEW

File: lib/migrations/migrate/stub/knexfile-ts.stub [hunk 1]
```diff
@@ -1,7 +1,8 @@
-// Update with your config settings.
+import type { Knex } from "knex";
 
-module.exports = {
+// Update with your config settings.
 
+const config: { [key: string]: Knex.Config } = {
```
Comment (kibertoad) [inline lib/migrations/migrate/stub/knexfile-ts.stub:5]: why [key: string]?

Comment (tharrison4815) [reply lib/migrations/migrate/stub/knexfile-ts.stub:5]: It's an object with strings as the property key names and config objects as the property values. Is that not what knex expects?

Comment (kibertoad) [reply lib/migrations/migrate/stub/knexfile-ts.stub:5]: isn't just Knex.Config sufficient?

Comment (kibertoad) [reply lib/migrations/migrate/stub/knexfile-ts.stub:5]: oh, wait, this is for knexfile. Right

**Review by kibertoad — APPROVED** [review]

**Comment by kibertoad** [comment]
Released in 1.0.1.

**Pull request merged**